### ISSUE TITLE
Feature request: reintroduce abillity to add custom decks

### ISSUE THREAD

**Issue by jacwah** [stevschmid/track-o-bot#185]
_Sorry if I am posting this on the wrong bug tracker, but I could not find one for the website._

Before Track-o-bot had standardised deck definitions, the user had to manually create definitions for each deck they wanted to track. I think supplying a standard set of common decks is great. However, when playing a deck not included in the set, this causes problems. It is not possible to view matchup statistics etc for non-standard decks since they are simply labeled under Other.

Having custom decks alongside the standard ones would be a solution to the problem described above. It is the best of both worlds as a first time user can jump right in, while an advanced user can squeeze the most out of Track-o-bot.

What do you think?

**Comment by Pipolaye** [comment]
I think this addition would be really nice, I would love to be able to track my Reno Mage deck =)

**Comment by Nickabilla** [comment]
Yes please :)

**Comment by jw5801** [comment]
+1

As more sets are released it is getting harder and harder to fit the decks around into the options available in a meaningful way.

**Comment by JustABanana** [comment]
Yes, we need this especially in un'goro! There are so many decks!

I know I'm late for a party

**Comment by nienkeboomsma** [comment]
Would love this feature. The current selection of decks is really out of date. Allowing users to type in custom deck archetypes would make this a moot point.
